forge-gui/res/cardsfolder/upcoming/nightkin_ambusher.txt

```diff
@@ -0,0 +1,10 @@
+Name:Nightkin Ambusher
+ManaCost:2 U B
+Types:Creature Mutant Warrior
+PT:4/4
+K:Ward:2
+T:Mode$ ChangesZone | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigRadiation | TriggerDescription$ When CARDNAME enters the battlefield, target player gets four rad counters.
+SVar:TrigRadiation:DB$ Radiation | ValidTgts$ Player | Add$ 4
+S:Mode$ CantBlockBy | ValidAttacker$ Card.Self | CheckSVar$ X | Description$ CARDNAME can't be blocked as long as defending player has a rad counter.
+SVar:X:TriggeredDefendingPlayer$Counters.RAD
```

Comment:
Don't know what the variable should be here

Reply:
try now

Reply:
Don't know if it's from my end but it does seem to crash now when it enters. 

I think PlayerCountDefinedDefendingPlayer is causing it ? 

![image](https://github.com/Card-Forge/forge/assets/67333662/44fdcbf0-554d-4b81-abb4-a0694f4076af)

Reply:
now?

Reply:
It doesn't crash anymore when it ETB's but it does crash at combat step 

```
ame-0 > java.lang.NullPointerException: Cannot invoke "forge.game.player.Player.getGame()" because "player" is null
	at forge.game.ability.AbilityUtils.playerXProperty(AbilityUtils.java:3324)
	at forge.game.ability.AbilityUtils.playerXCount(AbilityUtils.java:3312)
	at forge.game.ability.AbilityUtils.calculateAmount(AbilityUtils.java:527)
	at forge.game.ability.AbilityUtils.calculateAmount(AbilityUtils.java:398)
	at forge.game.staticability.StaticAbility.checkConditions(StaticAbility.java:446)
	at forge.game.staticability.StaticAbility.checkConditions(StaticAbility.java:302)
	at forge.game.staticability.StaticAbilityCantAttackBlock.cantBlockBy(StaticAbilityCantAttackBlock.java:149)
	at forge.game.combat.CombatUtil.canBlock(CombatUtil.java:1151)
	at forge.ai.AiAttackController.getCardCanBlockAnAttacker(AiAttackController.java:352)
	at forge.ai.AiAttackController.canBlockAnAttacker(AiAttackController.java:343)
	at forge.ai.AiAttackController$2.apply(AiAttackController.java:337)
	at forge.ai.AiAttackController$2.apply(AiAttackController.java:1)
	at com.google.common.collect.Iterators$5.computeNext(Iterators.java:673)
	at com.google.common.collect.AbstractIterator.tryToComputeNext(AbstractIterator.java:146)
	at com.google.common.collect.AbstractIterator.hasNext(AbstractIterator.java:141)
	at forge.util.collect.FCollection.addAll(FCollection.java:310)
	at forge.util.collect.FCollection.<init>(FCollection.java:90)
	at forge.game.card.CardCollection.<init>(CardCollection.java:115)
	at forge.game.card.CardLists.filter(CardLists.java:355)
	at forge.ai.AiAttackController.getPossibleBlockers(AiAttackController.java:334)
	at forge.ai.AiAttackController.refreshCombatants(AiAttackController.java:120)
	at forge.ai.AiAttackController.<init>(AiAttackController.java:92)
	at forge.ai.AiAttackController.<init>(AiAttackController.java:84)
	at forge.ai.AiController.declareAttackers(AiController.java:1304)
	at forge.ai.PlayerControllerAi.declareAttackers(PlayerControllerAi.java:644)
	at forge.game.phase.PhaseHandler.declareAttackersTurnBasedAction(PhaseHandler.java:571)
	at forge.game.phase.PhaseHandler.onPhaseBegin(PhaseHandler.java:309)
	at forge.game.phase.PhaseHandler.startFirstTurn(PhaseHandler.java:1147)
	at forge.game.GameAction.startGame(GameAction.java:2113)
	at forge.game.Match.startGame(Match.java:90)
	at forge.gamemodes.match.HostedMatch$2.run(HostedMatch.java:258)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)

```

Reply:
ugh I'll debug

Reply:
Thanks I think it's working now 

![image](https://github.com/Card-Forge/forge/assets/67333662/5f972f72-46d0-4743-9e90-d88f2401b29d)